torch/_dynamo/external_utils.py

```diff
@@ -27,3 +35,21 @@ def call_hook(hook, *args):
     if result is None:
         return args[0]
     return result
+
+
+def wrap_np(f):
```

Comment:
Should we call this wrap_numpy?  Or something more descriptive?  Wrap is pretty generic...

Reply:
Renamed to the more explicit `wrap_numpy`